package.json

```diff
@@ -19,6 +19,7 @@
     "test:server:up": "cd .. && PIPENV_DONT_LOAD_ENV=1 pipenv run test-server-up"
   },
   "dependencies": {
+    "@vue/composition-api": "^1.0.0-beta.16",
```

Comment:
Do you think we're ok to use this prerelease version?

Reply:
I believe there's some weirdness in the pre-beta versions, and they've been on betas for four months now. I'm personally okay with it, but we might want to pin this instead of allowing for minor and patch (and therefore any beta version) bumps.